splunk/splunk.go

```diff
@@ -17,3 +21,19 @@ func GetSplunkAppAPIEndpoint(host, port, ownerName, appName string) string {
 func GetSearchAPIEndpoint(appAPIEndpoint string) string {
 	return fmt.Sprintf("%s/search/v2/jobs/export", appAPIEndpoint)
 }
+
+//go:embed splunk_container_template.tmpl
+var splunkServiceManifest string
+
+const NotUpErrorMsg = "The %s instance is not up, cannot verify indexing for tests."
+
+// WithSplunkContainer builds a splunk container
```

Comment:
While it may need better documentation, I don't believe this line gives any more context than the function does by reading it.